exampleCourse/questions/element/dropdown/question.html

```diff
@@ -5,20 +5,20 @@
 </pl-question-panel>
 
 <p>
-  The 
-  <pl-dropdown answers-name="aristotle">
-    {{#params.aristotle}}
+  The
+  <pl-multiple-choice answers-name="aristotle" display="dropdown" hide-letter-keys="true">
+    {{#params.question1}}
         <pl-answer correct="{{tag}}">{{ans}}</pl-answer>
-    {{/params.aristotle}}
-  </pl-dropdown> 
+    {{/params.question1}}
+  </pl-multiple-choice>
   is more than the sum of its parts.
 </p>
 <p>
   A
-  <pl-dropdown sort="ascend" answers-name="hume">
+  <pl-multiple-choice sort="ascend" answers-name="hume" display="dropdown" hide-letter-keys="true">
```

Comment:
This question can't even generate successfully because the presence of the `sort` attribute here:

<img width="843" alt="Screenshot 2024-02-07 at 11 31 20" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/f66bb78c-3589-48a6-b797-8cba0fda1110">

Reply:
Changes to this question have since been reverted.